src/typemap.c

```diff
@@ -509,9 +509,19 @@ int jl_typemap_intersection_visitor(union jl_typemap_t map, int offs,
             ty = jl_tparam(ttypes, offs);
         }
         if (ty) {
+            while (jl_is_typevar(ty))
+                ty = ((jl_tvar_t*)ty)->ub;
+            // approxify the tparam until we have a valid type
```

Comment:
approxify is not a word unless you live in silicon valley.

Reply:
I didn't write that comment, it's a good ol' fashioned jamesonism